xcode/Subconscious/Shared/Models/Slug.swift

```diff
@@ -69,7 +76,22 @@ public struct Slug:
     
     /// Excludes "internal" slugs like `Slug.profile`
     public var isHidden: Bool {
-        verbatim.hasPrefix("_")
+        verbatim.hasPrefix(Self.hiddenPrefix)
+    }
+    
+    public init?(hidden: String) {
+        guard hidden.wholeMatch(of: Self.slugRegex) != nil else {
+            return nil
+        }
+        
+        // Consider the case where this slug is already prefixed
+        if hidden.starts(with: Self.hiddenPrefix) {
```

Comment:
Might be worth not being smart about this. Principle of least surprise. Consistent behavior may be preferable. (Non-blocking)